codespell_lib/data/dictionary.txt

```diff
@@ -38860,22 +38944,36 @@ vieports->viewports
 vietnamesea->Vietnamese
 viewtransfromation->viewtransformation
 vigeur->vigueur, vigour, vigor,
+vigilanties->vigilantes
+vigilanty->vigilante
 vigilence->vigilance
 vigourous->vigorous
 vill->will
+villan->villain
+villans->villains
 villian->villain
+villians->villains
 villification->vilification
 villify->vilify
 villin->villi, villain, villein,
+vinal->vinyl
```

Comment:
`vinal` appears to be an alternative spelling of `vinyl` as a synthetic textile fibre. Move to _rare_?

Reply:
Yes, done.